scripts/update-android-docs.sh

```diff
@@ -0,0 +1,113 @@
+#!/usr/bin/env bash
+
+# Requirements:
+#   - git
+#   - gh - GitHub's official command line tool [https://github.com/cli/cli].
+#   - jq - command-line JSON processor [https://github.com/stedolan/jq]
+
+JAVADOC_CMD="make android-javadoc"
```

Comment:
here and below `readonly` would fit well.

Reply:
Well, probably we have to make `JAVADOC_CMD`  and `JAVADOC_OUTPUT_DIR` overrideable at some point (with command line arguments), if we want to reuse the script for Carbon. But for others - maybe it make some sense, I don't know. :-D